packages/typespec-test/test/loadTest/generated/typespec-ts/src/outputModels.ts

```diff
@@ -381,7 +512,7 @@ export interface DimensionValueOutput {
 }
 
 export interface DimensionValueListOutput {
-  value: Record<string, any>;
+  value: string[];
```

Comment:
why?

Reply:
Currently we model anonymous model as `Record<string, any>`